Open Judge System/Web/OJS.Web/Scripts/Administration/Problems/problems-create.js

```diff
@@ -48,27 +77,40 @@ $(document).ready(function () {
         numericTextBox.enable(true);
         input.attr("data-val-required", "Лимита е задължителен!");
 
-        $("form").removeData("validator");
-        $("form").removeData("unobtrusiveValidation");
-        $.validator.unobtrusive.parse($('form'));
+        $("#create-form form").removeData("validator");
+        $("#create-form form").removeData("unobtrusiveValidation");
+        $.validator.unobtrusive.parse($('#create-form form'));
     }
 
     $('#enable-sclimit').change(function () {
         if ($(this).is(':checked')) {
             numericTextBox.enable(true);
             input.attr("data-val-required", "Лимита е задължителен!");
-            
-            $("form").removeData("validator");
-            $("form").removeData("unobtrusiveValidation");
-            $.validator.unobtrusive.parse($('form'));
+
+            $("#create-form form").removeData("validator");
+            $("#create-form form").removeData("unobtrusiveValidation");
+            $.validator.unobtrusive.parse($('#create-form form'));
         }
         else {
             numericTextBox.enable(false);
             input.removeAttr("data-val-required");
 
-            $("form").removeData("validator");
-            $("form").removeData("unobtrusiveValidation");
-            $.validator.unobtrusive.parse($('form'));
+            $("#create-form form").removeData("validator");
```

Comment:
Try to search for `$("#create-form form").removeData("validator");` in this file. 
Maybe we must extract these three rows in function?